### ISSUE TITLE
Is it possible to mount a "shared data" volume in addition to the user volume when using dockerspawner?

### ISSUE THREAD

**Issue by Alive24** [jupyterhub/dockerspawner#266]
I would like to share some datasets to all users when they start to use jupyterhub, and I would like to make these files read-only.  
I've considered using other ways of directly adding these datasets to these spawn volumes, but I wish to save the effort and disk usage of generating additional copies for all the spawned servers.

At the moment,  I'm trying to tweak `jupyterhub_config.py` this way:
```python
c.DockerSpawner.volumes = 
{'jupyterhub-user-{username}': notebook_dir, 
'test': os.path.join(notebook_dir, 'test)
}
```

However, I'm not familiar with docker volume mounting, particularly whether it would contaminate the user-specific volume if I mount another volume under the same location. Moreover, I would like to set the shared data as read-only.

Thank you!

**Comment by cmseal** [comment]
Hi @xct24,

I'm looking at similar things at the moment, and I've been able to get the mount to work this way.  Note that I've got the user-specific volume covering the whole of the jovyan home dir, not just 'work', but the same theory should apply.

Hope this helps... and maybe someone with more docker knowledge can chip in with the "mount inside a mount"/contamination aspect etc.

`c.DockerSpawner.volumes = { 
'jupyterhub-user-{username}':'/home/jovyan',  
'/path/to/shared': {"bind": '/home/jovyan/shared', "mode": "ro"}
  }`
